desc/compute/_metric.py

```diff
@@ -2088,3 +2088,562 @@ def _cvdrift0(params, transforms, profiles, data, **kwargs):
         * dot(data["b"], cross(data["grad(|B|)"], data["e^rho"]))
     )
     return data
+
+
+################################################################################
+##########-----------------METRIC ELEMENTS PEST----------------------###########
+################################################################################
+
+
+@register_compute_fun(
+    name="g_rr|PEST",
+    label="g_{\\rho\\rho}|PEST",
+    units="m^{2}",
+    units_long="square meters",
+    description="Radial/Radial element of covariant metric tensor" + "PEST_coordinates",
```

Comment:
```suggestion
    description="Radial/Radial element of covariant metric tensor" + " in PEST coordinates",
```

same for the others in this file (main thing is make sure to include the space, as is this will render as `metric tensorPEST_coordinates`